Examples/PublisherExampleSwiftUI/PublisherExampleSwiftUI/View/Settings/SettingsView.swift

```diff
@@ -64,6 +64,9 @@ struct SettingsView: View {
                 Toggle(isOn: $viewModel.areRawLocationsEnabled) {
                     Text("Raw locations enabled")
                 }
+                Toggle(isOn: $viewModel.useMapBox) {
+                    Text("Use MapBox")
```

Comment:
This text kinda implies we can turn off the whole Mapbox integration, while we're just toggling which map will be used. So maybe `Use MapBox map` instead?

Reply:
changed